### PR TITLE
AliEsmaeili tehran list

### PR DESCRIPTION
**Pull request by AliEsmaeili**
A preliminary list to eat in Tehran. I would update it after thinking a while.

### CODE REVIEW

**Comment by lumaquan** [comment]
What specialities are served in Masoodie?

File: AliEsmaeili-tehran.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Tehran, Tehran
+
+## Places to Eat
+- Tehroon Cafe
```
Comment (mohitspawar) [inline AliEsmaeili-tehran.md]: What kind of food does Tehroon Cafe serve ?

**Review by mohitspawar — COMMENTED** [review]
Can you add what kind of cuisine each place serves ?

File: AliEsmaeili-tehran.md (hunk 1 above)
Comment (AliEsmaeili) [reply AliEsmaeili-tehran.md]: Really they have changed place recently, but they have good persian traditional food.

**Comment by AliEsmaeili** [comment]
check it here: [Masoodeyeh Cafe](https://foursquare.com/v/masoudiyeh-caf%C3%A9--%DA%A9%D8%A7%D9%81%D9%87-%D9%85%D8%B3%D8%B9%D9%88%D8%AF%DB%8C%D9%87/537eedf8498e635a12b8dab1)

They also have very good brewing teas.

**Review by lumaquan — APPROVED** [review]
Excellent!!

**Pull request merged**